Bottom: 2f95dab57b18d062043ebc4da91a2c2b51f087ec
Top:    77334935921eeb9bf6211781066d1e5dbdf3da81
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 16:55:10 -0500

Refresh of sync-bacula-restores

---
diff --git a/roles/bacula-restores/defaults/main.yml b/roles/bacula-restores/defaults/main.yml
new file mode 100644
index 00000000..b72968f6
--- /dev/null
+++ b/roles/bacula-restores/defaults/main.yml
@@ -0,0 +1,3 @@
+---
+
+bacula_storage_path: "/mnt/bacula-storage"
diff --git a/roles/bacula-restores/tasks/main.yml b/roles/bacula-restores/tasks/main.yml
new file mode 100644
index 00000000..1e614c88
--- /dev/null
+++ b/roles/bacula-restores/tasks/main.yml
@@ -0,0 +1,11 @@
+---
+# tasks file for bacula-restores
+
+- name: Synchronize bacula storage to restore directory
+  synchronize:
+    dest: "{{bacula_dest}}/" # required. Path on the destination host that will be synchronized from the source; The path can be absolute or relative.
+    src: "{{bacula_storage_path}}/" # required. Path on the source host that will be synchronized to the destination; The path can be absolute or relative.
+    delete: yes # not required. Delete files in C(dest) that don't exist (after transfer, not before) in the C(src) path. This option requires C(recursive=yes).
+    archive: yes # not required. Mirrors the rsync archive flag, enables recursive, links, perms, times, owner, group flags and -D.
+    link_dest: "{{bacula_storage_path}}/" # not required. add a destination to hard link against during the rsync.
+  delegate_to: "{{ inventory_hostname }}"
